grpc/py-cubic/cubic/client.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(2019) Cobalt Speech and Language Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http: // www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import grpc
+
+from google.protobuf.empty_pb2 import Empty
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__))))
```

Comment:
It's a long winded way of adding a directory pointed to by `__file__` to the import path.
`__file__` = pathname of the file from which the module was loaded.

It's needed so that the protoc generated files can find each other